tests/test_middleware.py

```diff
@@ -113,3 +113,28 @@ async def lifespan_only_app(scope: Scope, receive: Receive, send: Send) -> None:
     app = MessagePackMiddleware(lifespan_only_app)
     scope = {"type": "lifespan"}
     await app(scope, mock_receive, mock_send)
+
+
+@pytest.mark.asyncio
+async def test_packb_unpackb() -> None:
+    async def app(scope: Scope, receive: Receive, send: Send) -> None:
+        request = Request(scope, receive)
+        response = JSONResponse({"message": "Hello, World!"})
+
+        await response(scope, receive, send)
+        assert "unpacked" == await request.json()
+
+    app = MessagePackMiddleware(
+        app, packb=lambda obj: b"packed", unpackb=lambda byt: "unpacked"
```

Comment:
(Nit) I think the raw JSON string is a bit jarring — should we return a dict, which seems more usual?

```suggestion
        app, packb=lambda obj: b"packed", unpackb=lambda byt: {"message": "unpacked"}
```